src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/ReportsService.java

```diff
@@ -59,13 +59,26 @@ public ReportsService(
         return reportResult;
     }
 
+    public int getTotalReceived(List<EnvelopeCountSummary> items) {
+        return items.stream().mapToInt(o -> o.received).reduce(0, (a, b) -> a + b);
+    }
+
+    public int getTotalRejected(List<EnvelopeCountSummary> items) {
+        return items.stream().mapToInt(o -> o.rejected).reduce(0, (a, b) -> a + b);
+    }
+
+    public String getTimeStamp() {
```

Comment:
we do not need these methods. (or just make them private) you have final class fields. You can initiliaze them in constructor